clang/test/CodeGenSYCL/simplifycfg.cpp

```diff
@@ -1,5 +1,4 @@
-// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -mllvm -sycl-opt %s -emit-llvm  -o - | FileCheck %s
-// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -mllvm -sycl-opt %s -emit-llvm -o - | FileCheck %s
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -mllvm -sycl-opt %s -emit-llvm -O3 -o - | FileCheck %s
```

Comment:
Ok, I understood reasoning for -O1, but -O3 is a higher level of optimization than -O2 (previous default) meaning it probably has more passes executed, so this likely doesn't make the test faster. Why is it used here now and not -O2?

Reply:
I read the comment on line 3, which says that this test is supposed to check LLVM IR "after O3 optimizations".
As you can see the test has two exactly the same "run" lines. I suppose that one of them were previously set O3, but it was removed for some reason.